### PR TITLE
Pre-commit update

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Update pre-commit tools version. Because of pydantic use then new option for `pyupgrade` is added

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: napari/layers/vectors/vectors.py [hunk 1]
```diff
@@ -573,7 +573,7 @@ def edge_contrast_limits(
 
     @property
     def _view_face_color(self) -> np.ndarray:
-        """" (Mx4) np.ndarray : colors for the M in view vectors"""
+        """ " (Mx4) np.ndarray : colors for the M in view vectors"""
```
Comment (tlambert03) [inline napari/layers/vectors/vectors.py]:
```suggestion
        """(Mx4) np.ndarray : colors for the M in view vectors"""
```

**Review by tlambert03 — COMMENTED** [review]
thanks!  looks mostly good... but something is touching the vendored files.  (for f-strings, and inheriting from `object`).  do you know whether thats pyupgrade? (and if so, can you see if our `exclude` string needs updating?

**Comment by Czaki** [comment]
> thanks! looks mostly good... but something is touching the vendored files. (for f-strings, and inheriting from `object`). do you know whether thats pyupgrade? (and if so, can you see if our `exclude` string needs updating?

this is my bad with exclusion when I try to check. I will revert this.

**Review by sofroniewn — APPROVED** [review]
Changes here look good to me - thanks for undo vendored changes, I'll leave open over night in case any other folks want to weigh in, but will then get this in as it touches a lot of files!!

**Review by tlambert03 — APPROVED** [review]

**Pull request merged**